sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParser.java

```diff
@@ -131,4 +147,79 @@ private void handleSegmentPointTag(XmlParserHelper xmlParserHelper) {
 
   }
 
+  /**
+   * This class is useful when on a single line there are multiple methods. Some may be covered, some not
```

Comment:
I think this message is misleading. This class actually aggregates all the overlapping segments for a line together with their hits.